Minor tail work to add missing documentation for CarPlay changes in last commit (#2917)
mapbox/mapbox-navigation-ios: CHANGELOG.md

* Added the `NavigationViewController.styleManager`, `StyleManager.currentStyleType`, and `StyleManager.currentStyle` properties and the `StyleManager.applyStyle(type:)` method to manually change the UI style at any time. ([#2888](https://github.com/mapbox/mapbox-navigation-ios/pull/2888))
* Fixed crash when switching between day / night modes ([#2896](https://github.com/mapbox/mapbox-navigation-ios/pull/2896))
* Add support for iOS 13's UIScene based CarPlay API ([#2832](https://github.com/mapbox/mapbox-navigation-ios/pull/2832))

> * Added new CarPlayManagerDelegate method  `CarPlayManagerDelegate.carPlayManager(_:didPresent:)`  used when the CarPlayManager presents a new navigation session ([#2832](https://github.com/mapbox/mapbox-navigation-ios/pull/2832))

Reviewer: ```suggestion
* Added an optional `CarPlayManagerDelegate.carPlayManager(_:didPresent:)` method that is called when `CarPlayManager` presents a new navigation session. ([#2832](https://github.com/mapbox/mapbox-navigation-ios/pull/2832))
```
